ialirt_data_access/cli.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python3
+
+"""Command line interface to query IALIRT logs in the s3 bucket.
+
+Usage:
+    ialirt_data_access --debug --url <url> ialirt-log-query
+    --year <year> --doy <doy> --instance <instance>
+"""
+
+import argparse
+import logging
+
+import ialirt_data_access
+
+
+def _query_parser(args: argparse.Namespace):
+    """Query the IALIRT log API.
+
+    Parameters
+    ----------
+    args : argparse.Namespace
+        Parsed arguments including year, doy, and instance.
+
+    Returns
+    -------
+    None
+    """
+    query_params = {
+        "year": args.year,
+        "doy": args.doy,
+        "instance": args.instance,
+    }
+    try:
+        query_results = ialirt_data_access.query(**query_params)
+        print(query_results)
+    except ialirt_data_access.io.IALIRTDataAccessError as e:
+        print(e)
+        return
+
+
+def main():
+    """Parse the command line arguments.
+
+    Run the command line interface to the IALIRT Data Access API.
+    """
+    url_help = (
+        "URL of the IALIRT API. "
+        "The default is https://ialirt.dev.imap-mission.com. This can also be "
+        "set using the IALIRT_DATA_ACCESS_URL environment variable."
+    )
+
+    parser = argparse.ArgumentParser(prog="ialirt-data-access")
+    parser.add_argument(
+        "--version",
+        action="version",
+        version=f"%(prog)s {ialirt_data_access.__version__}",
+    )
+    parser.add_argument("--url", type=str, required=False, help=url_help)
+    # Logging level
+    parser.add_argument(
+        "--debug",
+        help="Print lots of debugging statements",
+        action="store_const",
+        dest="loglevel",
+        const=logging.DEBUG,
+        default=logging.WARNING,
+    )
+    parser.add_argument(
+        "-v",
+        "--verbose",
```

Comment:
What is the difference between "debug" and "verbose" here? Could you do `-v` for some verbosity and `-vv` for more?